plugins/tiledb/io/TileDBReader.cpp

```diff
@@ -105,6 +105,13 @@ void TileDBReader::addArgs(ProgramArgs& args)
     args.add("stats", "Dump TileDB query stats to stdout", m_stats, false);
     args.add("bbox3d", "Bounding box subarray to read from TileDB in format "
         "([minx, maxx], [miny, maxy], [minz, maxz])", m_bbox);
+    args.add("end_timestamp", "TileDB array timestamp", m_endTimeStamp,
+        point_count_t(0));
+    args.addSynonym("end_timestamp", "timestamp");
```

Comment:
Here `timestamp` is made a synonym for `end_timestamp`, but the doc text makes it seem as if it's a synonym for `start_timestamp`, but this is also tied to the TileDB version somehow. Maybe the doc just needs some clarification?

Reply:
Agreed, I went back and forth on how to name the time interval. I have updated the docs.